packages/button/src/vaadin-button.js

```diff
@@ -52,6 +52,14 @@ class Button extends ControlStateMixin(ActiveMixin(ElementMixin(ThemableMixin(Po
           padding: 0;
           border: none;
           box-shadow: none;
+          outline: none;
+          font-family: inherit;
+          font-size: inherit;
+          font-weight: inherit;
+          line-height: inherit;
```

Comment:
Could be simplified:

```suggestion
          font: inherit;
```

Reply:
Good catch, thanks! Applied.